.github/workflows/email-check.yaml

```diff
@@ -4,6 +4,7 @@ on:
   pull_request:
     types:
       - opened
+      - reopened
```

Comment:
Note for reviewers: I added this just to test this PR. Will remove it once everything else looks good.